index.yaml

```diff
@@ -54,4 +54,16 @@ projects:
     - neeraj.laad@gmail.com
     urls:
     - https://github.com/appsody/stacks/releases/download/nodejs-express-v0.2.1/incubator.nodejs-express.templates.simple.tar.gz
-    version: 0.2.1
\ No newline at end of file
+    version: 0.2.1
+  swift:
+  - created: 2019-07-15T12:00:00+00:00
+    description: Swift application
```

Comment:
I think we chose to go for Swift runtime in the website PR, similar to the Node.js stack

Reply:
good catch.. fixed.